src/coscale/api/alert.go

```diff
@@ -267,3 +278,75 @@ func ParseHandle(handle string) (string, error) {
 	jsonHandle, err := json.Marshal(result)
 	return string(jsonHandle), err
 }
+
+// ParseDimensionSpecs is used to parse the dimensions specs for a metric.
+func ParseDimensionSpecs(format string) (string, error) {
```

Comment:
Can we use the same format for the dimension specs as we use for retrieving data ?